deps.bzl

```diff
@@ -2205,8 +2205,8 @@ def install_go_mod_dependencies(workspace_name = "buildbuddy"):
     go_repository(
         name = "com_github_google_go_containerregistry",
         importpath = "github.com/google/go-containerregistry",
-        sum = "h1:MMkSh+tjSdnmJZO7ljvEqV1DjfekB6VUEAZgy3a+TQE=",
-        version = "v0.15.2",
+        sum = "h1:B1wPJ1SN/S7pB+ZAimcciVD+r+yV/l/DSArMxlbwseo=",
```

Comment:
nit: are the version bumps needed here?

Reply:
No, just something I tried hoping it would resolve the problem. I'll revert it.